csharp/config/suites/lgtm/csharp-alerts-lgtm

```diff
@@ -0,0 +1,310 @@
+# Auto-generated by:
+# ./build target/general/lgtm-query-suites/output/csharp-alerts-lgtm
```

Comment:
External contributors can't generate this file, and its contents may change depending on the tools in our internal repo. When we moved the lgtm suite to this repo for C++ and JavaScript, we made this file empty and changed the dist building rules to generate it at every dist build.

Reply:
Ah, good to know. @calumgrant: Will you make a follow-up PR?